osu.Game/Users/UpdateableAvatar.cs

```diff
@@ -47,6 +47,15 @@ protected override void LoadComplete()
             updateAvatar();
         }
 
+        /// <summary>
+        /// Clears the user instantly
+        /// </summary>
+        public void ClearUser()
+        {
+            user = null;
+            displayedAvatar?.FadeOut().Expire();
```

Comment:
the FadeOut here is unnecessary i think, unless you are cancelling a fadein?

Reply:
this shouldn't be reviewed since it's outdated, the conflict must be resolved first.

Reply:
It's still useful feedback on the changes I made. I can keep this in mind when redoing it on latest master.